src/main/java/mate/academy/springboot/dto/response/CategoryResponseDto.java

```diff
@@ -0,0 +1,11 @@
+package mate.academy.springboot.dto.response;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
```suggestion
@Data
```
you can use Data annotations - it has a lot of methods underneath